docs/ref/models/querysets.txt

```diff
@@ -1680,6 +1680,14 @@ whenever a request to a page has a side effect on your data. For more, see
   chapter because it isn't related to that book, but it can't create it either
   because ``title`` field should be unique.
 
+.. warning::
```

Comment:
I think we don't need a warning directive for this kind of information. You can add a separate versionchanged directive.